File: 강병수/C#/210215/class_Review2/class_Review2/StudentScore.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace class_Review2
{
    class StudentScore
    {
        private int kor;
        private int eng;
        private int math;
        private int sum;
        private double avr;
        private char grade;

        public StudentScore() { }

        public StudentScore(int kor, int eng, int math)
        {
            this.kor = kor;
            this.eng = eng;
            this.math = math;
        }

        public int Kor { get => kor; set => kor = value; }
        public int Eng { get => eng; set => eng = value; }
        public int Math { get => math; set => math = value; }
        public int Sum { get => sum; set => sum = value; }
        public double Avr { get => avr; set => avr = value; }
        public char Grade { get => grade; set => grade = value; }

        public void calculator()
        {
            sum = kor + eng + math;
            avr = (double)sum / 3;
            if (avr >=90 && avr <=100)
            {
                grade = 'A';
            }
            else if (avr >= 80 && avr < 90)
            {
                grade = 'B';
            }
            else if (avr >= 70 && avr < 80)
            {
                grade = 'C';
            }
            else if (avr >= 60 && avr < 70)
            {
                grade = 'D';
            }
            else if (avr < 60)
            {
                grade = 'F';
            }

            Console.WriteLine("총점: " + sum);
            Console.WriteLine("평균: " + avr);
            Console.WriteLine("학점: " + grade);
            Console.WriteLine("======================");
        }

        public void calculator(int kor, int eng, int math)
        {
            sum = kor + eng + math;
            avr = (double)sum / 3;
            if (avr >= 90 && avr <= 100)
            {
                grade = 'A';
            }
            else if (avr >= 80 && avr < 90)
            {
                grade = 'B';
            }
            else if (avr >= 70 && avr < 80)
            {
                grade = 'C';
            }
            else if (avr >= 60 && avr < 70)
            {
                grade = 'D';
            }
            else if (avr < 60)
            {
                grade = 'F';
            }

            Console.WriteLine("총점: " + sum);
            Console.WriteLine("평균: " + avr);
            Console.WriteLine("학점: " + grade);
            Console.WriteLine("======================");
        }

        public void setProperty(int kor, int eng, int math)
        {
            this.kor = kor;
            this.eng = eng;
            this.math = math;
        }
    }
}

File: 강병수/Java/210112/src/Problem4/StudentScore.java
package Problem4;

public class StudentScore {

	int sum;
	double avr;
	int [][] arr;
	int [] Sum;
	double [] Avr;
	char [] Grade;

	public StudentScore(int[][] arr) {
		this.arr = arr;
		Sum();
		Avr();
		Grade();
	}

	public void Sum() {
		for (int i = 0; i < arr.length; i++) {
			for (int j = 0; j < arr[i].length; j++) {
				sum += arr[i][j];
				Sum[i] = sum;
			}
			System.out.printf("학생%d의 총점: %d\n", i, Sum[i]);
			sum = 0;
		}
	}

	public void Avr() {
		for (int i = 0; i < arr.length; i++) {
			for (int j = 0; j < arr[i].length; j++) {
				avr = (double) Sum[i] / 3;
				Avr[i] = avr;
			}
			System.out.printf("학생%d의 평균: %d\n = ", i, Avr[i]);
			avr = 0;
		}
	}

	public void Grade() {
		for (int i = 0; i < arr.length; i++) {
			if (avr >= 90 && avr <= 100) {
				Grade[i] = 'A';
			} else if (avr >= 80 && avr < 90) {
				Grade[i] = 'B';
			} else if (avr >= 70 && avr < 80) {
				Grade[i] = 'C';
			} else if (avr >= 60 && avr < 70) {
				Grade[i] = 'D';
			} else if (avr >= 50 && avr < 60) {
				Grade[i] = 'E';
			} else if (avr >= 0 && avr < 50) {
				Grade[i] = 'F';
			}
			System.out.printf("학생%d의 총점: %s\n", i, Grade[i]);
			System.out.println("ㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡㅡ");
		}

	}
}